test/funnel.html

```diff
@@ -19,141 +18,196 @@
 -->
 
 <html>
-    <head>
-        <meta charset="utf-8">
-        <script src="lib/simpleRequire.js"></script>
-        <script src="lib/config.js"></script>
-        <script src="lib/dat.gui.min.js"></script>
-    </head>
-    <body>
-        <style>
-            html, body, #main {
-                width: 100%;
-                height: 100%;
-                margin: 0;
-            }
-        </style>
-        <div id="main"></div>
-        <script>
 
-            require([
-                'echarts'
-            ], function (echarts) {
+<head>
+    <meta charset="utf-8">
```

Comment:
Please add create a new test case here. Changing the original test case will break the automatic test